index.js

```diff
@@ -28,7 +28,13 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let arr = [];
+  for (let i = 0; i < movies.length; i++){
```

Comment:
Since we're not using `i` for anything except accessing an element in `movies`, we should be using a `for...of` loop here.